src/components/Splash.vue

```diff
@@ -160,14 +157,25 @@ export default defineComponent({
       startButtonTextConfig,
       isSessionDataFetched,
       startButtonIconConfig,
-      preventScrolling,
       start,
     };
   },
   emits: ["start"],
 });
 </script>
 
+methods: {
+  scrollContent(event) {
```

Comment:
what does this method do? can you add a comment above this method explaining this?